src/main/data/facilities.json

```diff
@@ -1539,6 +1539,16 @@
       "DLX"
     ]
   },
+  {
+    "code": "DAV",
+    "contact": {
+      "address": "Heroiv Dnepra 62, apt. 250, Ukraine, Kyev",
```

Comment:
Should this be "Kyiv"?

Reply:
The entry was Kyev, but I agree it's probably a mistype. I updated.